src/files.ts

```diff
@@ -302,6 +302,10 @@ export async function prepareExtendingBundle(
 
   if (processingFiles.length) {
     // Determine existing files (minus removed)
+    if (isWindows) {
```

Comment:
suggestion: @michelkaporin do you think it would make sense to write a test case for this (simulating the `isWindows` paramteter)?

Reply:
It's difficult since current tests are dependent on environment when working with bundle from what I see.
As mentioned in the PR description I think we should simply have a CircleCI agent that runs our tests under Windows environment. 

In fact [this test](https://github.com/snyk/code-client/blob/a4ffee2193889ddf8e408a247b66cfe4c37df53a/tests/files.spec.ts#L57) should have picked up the problem in my understanding, if the test would be run on Windows machine.

Reply:
@michelkaporin hmm, that makes sense. Ok, we might explore it later but won't be blocking you over this.